crates/duty-tracker/src/contract_state_machine.rs

```diff
@@ -746,6 +746,16 @@ impl ContractSM {
         (contract_sm, duty)
     }
 
+    /// Gets the configuration of the contract.
+    pub fn get_cfg(&self) -> &ContractCfg {
```

Comment:
I don't like the naming of `get` anywhere. We also have these functions already: `ContractSM::cfg` and `ContractSM::state`